### ISSUE TITLE
UI Redirect / to /ui/

### ISSUE THREAD

**Issue by joshuagiggxon** [distribworks/dkron#927]
**Describe the bug**
Before the UI got replaced with the React version, / would redirect to /dashboard/

Now, / 404's

Can it redirect again?

**Comment by yvanoers** [comment]
Yep that's possible, and it's one of the fixes in PR #926 (commit da5e32c).
